### PR TITLE
Task #79, #74, #78, #77 --> Revamp UI

### PR DESCRIPTION
**Pull request by rkhatta1**
Work DONE: Base UI and color scheme solidified.
Work Pending: Fix the Tabulation for Burndown Charts.

### CODE REVIEW

File: taigaProject/backend/taigaApi/authenticate.py [hunk 1]
```diff
@@ -26,5 +26,7 @@ def authenticate(username, password):
 
     except requests.exceptions.RequestException as e:
         # Handle authentication failure and print an error message
+        
+        print(f"{auth_token}")
```
Comment (rnolas96) [inline taigaProject/backend/taigaApi/authenticate.py:30]: Is this a debug statement? Please remove it.

File: taigaProject/frontend/src/App.js [hunk 2]
```diff
@@ -18,6 +19,9 @@ function App() {
   return (
     <div className="container-full">
       <Router>
+        <ProSidebarProvider>
+          <SidebarMenu />
```
Comment (rnolas96) [inline taigaProject/frontend/src/App.js:23]: Has the relative behavior between the SidebarMenu and the contents of each page been tested? I see it has been moved from within each component to the outside.

**Review by rnolas96 — COMMENTED** [review]
Looks mostly fine. Please address the comments.

**Comment by rnolas96** [comment]
Please fix the conflicts before merging as well. Take careful consideration not to remove any existing or recently modified code.

File: taigaProject/frontend/src/App.js (hunk 2 above)
Comment (rkhatta1) [reply taigaProject/frontend/src/App.js:23]: It has. For our relatively simple use-case scenarios, it most definitely will never create any issues.

**Pull request merged**